Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Convert.swift

```diff
@@ -434,42 +434,34 @@ extension Docc {
             @Flag(help: "Allows for custom templates, like `header.html`.")
             var experimentalEnableCustomTemplates = false
             
-            @Flag(help: .hidden)
-            @available(*, deprecated, message: "Objective-C support is enabled by default.")
-            var enableExperimentalObjectiveCSupport = false
-            
-            @Flag(help: .hidden)
-            @available(*, deprecated, message: "Render Index JSON is emitted by default.")
-            var enableExperimentalJSONIndex = false
-            
             @Flag(help: .hidden)
             var enableExperimentalDeviceFrameSupport = false
             
             @Flag(help: "Inherit documentation for inherited symbols")
             var enableInheritedDocs = false
-            
+
             @Flag(help: .hidden)
-            @available(*, deprecated, message: "Doxygen support is now enabled by default.")
+            @available(*, deprecated, message: "Doxygen support is now enabled by default. This deprecated API will be removed after 5.10 is released")
             var experimentalParseDoxygenCommands = false
-            
+        
             @Flag(help: "Experimental: allow catalog directories without the `.docc` extension.")
             var allowArbitraryCatalogDirectories = false
-            
+        
             @Flag(help: "Write additional metadata files to the output directory.")
             var emitDigest = false
-            
+        
             @Flag(
                 help: ArgumentHelp(
                     "Writes an LMDB representation of the navigator index to the output directory.",
                     discussion: "A JSON representation of the navigator index is emitted by default."
                 )
             )
             var emitLMDBIndex = false
-            
+
             @Flag(help: .hidden)
             @available(*, deprecated, renamed: "emitLMDBIndex")
             var index = false
-            
+        
```

Comment:
NIT:
Is there a format tool being used? Maybe we should leave it as is. (Someone removes the tab/space in one PR and someone adds it back in another PR. 🤣)

Or add your format/lint configuration into the repo so that everyone can follow it later.